dashboard/client/src/pages/metrics/Metrics.tsx

```diff
@@ -358,13 +390,25 @@ export const Metrics = () => {
 
   const grafanaDefaultDatasource = dashboardDatasource ?? "Prometheus";
 
+  const [refreshOption, setRefreshOption] = useState<RefreshOptions>(
+    RefreshOptions.ONE_SECOND,
```

Comment:
Can we make the default five seconds? one second seems too frequent.

Reply:
Sure! Five seconds is also the frequency we prefer. I was worried that changing from continuous updates to five seconds would change the habits of historical users. Seems like an unnecessary worry.